drift/info_parser.py

```diff
@@ -187,6 +184,12 @@ def _group_comparisons(comparisons):
         else:
             grouped_comparisons.append(comparison)
 
+    # sorting comparisons if exists
+    for comparison_name in grouped_comparisons_with_dotnotation:
+        group = grouped_comparisons_with_dotnotation.get(comparison_name)
+        group["comparisons"] = sorted(
+            group["comparisons"], key=lambda comparison: comparison_name
```

Comment:
CI fails here. Check where the `comparison_name` comes from within the lambda and you'll find a solution.